fabric-networking-entity-v1/src/main/java/net/fabricmc/fabric/api/networking/entity/v1/EntityNetworking.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.networking.entity.v1;
+
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.EntityType;
+import net.minecraft.network.Packet;
+
+import net.fabricmc.fabric.impl.networking.entity.v1.FabricEntityNetworking;
+
+/**
+ * Fabric entity networking hooks.
+ */
+public interface EntityNetworking {
```

Comment:
`EntityNetworking` doesn't sound like a class that has instances. I'd either rename it to `EntityNetworkingHelper` like [ResourceManagerHelper](https://github.com/FabricMC/fabric/blob/1.14/fabric-resource-loader-v0/src/main/java/net/fabricmc/fabric/api/resource/ResourceManagerHelper.java), or replace the implementation with delegated static methods like [OverworldBiomes](https://github.com/FabricMC/fabric/blob/1.14/fabric-biomes-v1/src/main/java/net/fabricmc/fabric/api/biomes/v1/OverworldBiomes.java)

Reply:
`EntitySpawnPacketHelper`?

Reply:
`EntitySpawnPacketHelper` sounds good to me!